src/k8s/deployment.ts

```diff
@@ -71,6 +75,15 @@ export class DeploymentConfig {
             'You have no replicas available');
     }
 
+    static async showLog(context: { id: any; }): Promise<string> {
+        let deployName: string = context ? context.id : null;
```

Comment:
`context.id => context.name` otherwise it is asking to select deployment config from quick pick.